### DISCUSSION TITLE
Dart/Flutter port

### DISCUSSION THREAD

**Discussion by encrystudio** [Ant-Brain/EfficientWord-Net#54] [General]
Hi there!
I really like this project and used it for some projects - never disappointed. However, developing an actual app with it is difficult with python. I think a dart package would make it a lot easier. Do you think that this is even possible? There are onnx and tflite runtime packages for dart, so essential packages can be replaced. I'm not a dart expert, but I'd like to try a port. Do you think it's possible?

And will there be further updates for this repo? I've seen so many open source ai/ml projects that where abandoned at some point because of big tech developing proprietary coud solutions. This is sad since many open source projects have potential to be better than closed software, even (or especially) ai and ml like you can see with this repo

So yeah would be cool to see some activity here, maybe I can help somehow?

**Comment by TheSeriousProgrammer** [comment]
Thank you for the kind words, currently working on a binary quantization approach for another project of mine with tflite, once I stream line it ,the same would be also be implemented in this project (that's the reason behind recent inactivity of this project)

Once it's ported to tflite, we could pretty much use it almost any development tool out there, sat tuned

**Reply by encrystudio** [reply]
Nice, I'm already hyped xD
